YesGraph/python-yesgraph
# Add/fix tests

#### What does this PR do?

Adds tests for all endpoints, making sure we're preparing our requests in the correct fashion. I've made some minor changes to the client along the way (renaming arguments, making some optional, etc.)

#### How should this be manually tested?

Simply run `tox`. It should not make any actual HTTP requests, so the API service does not have to be running for this to work.
